clinical/src/main/kotlin/com/hartwig/actin/clinical/feed/standard/DataQualityMask.kt

```diff
@@ -44,9 +60,12 @@ fun ProvidedMolecularTestResult.isAllFieldsExceptGeneNull(): Boolean {
             this.vaf == null
 }
 
-class DataQualityMask(private val panelGeneList: PanelGeneList) {
+class DataQualityMask(private val panelGeneList: PanelGeneList, private val clinicalConfiguration: ClinicalConfiguration) {
     fun apply(ehrPatientRecord: ProvidedPatientRecord): ProvidedPatientRecord {
-        return ehrPatientRecord.scrubMedications().scrubModifications().addAlwaysTestedGenes(panelGeneList)
+        return ehrPatientRecord.scrubMedications()
+            .scrubModifications()
+            .addAlwaysTestedGenes(panelGeneList)
             .removeAllEmptyMolecularTestResults()
+            .useOnlyPriorOtherConditions(clinicalConfiguration)
```

Comment:
The name of this function suggests that only POCs will be used in all cases. I think it would be better to define e.g. `scrubbed` from the above and then:
```suggestion
            return if (clinicalConfiguration.useOnlyPriorOtherConditions) {
                scrubbed.useOnlyPriorOtherConditions()
            } else scrubbed
```